canonical-websites/www.ubuntu.com
# Added canonical-global-nav as a dependency

## Done
Added canonical-global-nav as a dependency and removed all the old global navigation code.

## QA
- Pull branch and run `./run`
- Go to http://0.0.0.0:8001/
- See that the global navigation is there and works
- Check small screens

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/1725 https://github.com/canonical-websites/www.ubuntu.com/issues/1687

Demo: http://www.ubuntu.com-global-nav.demo.haus
